build/bin/remove_unneeded

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
nit, consider renaming `remove_unneeded` to `remove_unneeded.sh` - the suffix is a useful hint.

Reply:
Also, if you don't already, please make sure to remove `remove_unneeded.sh` from the scratch image.

Reply:
It is not copied to scratch.